File: Api/Responses/Game/RegisterPlayerResponse.cs
namespace OhMyWord.Api.Responses.Game;

public class RegisterPlayerResponse
{
    public bool RoundActive { get; init; }
    public int PlayerCount { get; init; }
    public long Score { get; init; }
    public RoundStartResponse? RoundStart { get; init; }
    public RoundEndResponse? RoundEnd { get; init; }
}

File: Client/src/app/models/responses/register-player.response.ts
import { RoundEndResponse } from './round-end.response';
import { RoundStartResponse } from './round-start.response';

export interface RegisterPlayerResponse {
  roundActive: boolean;
  playerCount: number;
  score: number;
  roundStart: RoundStartResponse | null;
  roundEnd: RoundEndResponse | null;
}
